Entities/Meta/ChallengeMusic.as

```diff
@@ -95,15 +97,23 @@ void GameMusicLogic(CBlob@ this, CMixer@ mixer)
 		if (timer % 48 != 0)
 			return;
 
+		bool isNight = map.getDayTime() > 0.75f;
```

Comment:
This seems to be a generally poor value. `>0.85 && <0.1` would probably be more fitting with the current sky gradient.

Reply:
```suggestion
		bool isNight = map.getDayTime() > 0.85f && map.getDayTime() < 0.1f;
```
I have now realized that fix suggestions are a thing.